controllers/pipelines/provider_controller.go

```diff
@@ -248,20 +251,39 @@ func populateServiceContainer(serviceContainerName string, podTemplate v1.PodTem
 		return nil, fmt.Errorf("unable to populate service container: container with name %s not found on deployment", serviceContainerName)
 	}
 
+	var paramsAsEnvVars []v1.EnvVar
```

Comment:
```suggestion
	paramsAsEnvVars := []v1.EnvVar{}
```

Reply:
Done in 12dd027 (fixed the length as well)